repository/src/test/java/org/alfresco/repo/rendition2/AbstractRenditionIntegrationTest.java

```diff
@@ -298,6 +309,7 @@ protected void clearContent(String user, NodeRef sourceNodeRef)
                 transactionService.getRetryingTransactionHelper().doInTransaction(() ->
                 {
                     clearContent(sourceNodeRef);
+                    clearRendition(sourceNodeRef);
```

Comment:
IIRC, we don't currently have a remote public API to clear renditions (maybe it would be useful, eg. admin-only) ?

Hence, why do we need to test here ? Would it be better to create a new node &/or version ?